backend/app/services/adapters/subtask_formatter.py

```
@@ -0,0 +1,1091 @@
+        if not isinstance(private_model_name, str) or not private_model_name.strip():
+            return agent_config
+
+        try:
+            model_name = private_model_name.strip()
+            public_model = db.query(Kind).filter(Kind.name == model_name).first()
+
+            if public_model and public_model.json:
+                model_config = public_model.json.get("spec", {}).get("modelConfig", {})
+                return model_config
+
+        except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)